### ISSUE TITLE
Is there a Stop Function

### ISSUE THREAD

**Issue by bytenaija** [locize/locizify#159]
When you set autorun: false it gives you an object with a start function. Is there a stop function? We are trying to use locizify to auto-translate only one page in React. 

Using i18next will mean using the Trans component and we would have to manually enter hundreds if not thousands of keys to the i18nKey props. We are trying to avoid this by using locizify.

We observed that translations is currently bleeding into multiple components whereas we would like to restrict it to a single component say `ArtistComponent`. Using `start` helps to start the translation, bu we would like the option of stopping when a user moves away from that component.

**Comment by jamuhl** [comment]
No...the point is...do not use locizify with react...it is really not the use case it was created for.

Under the hood locizify (using i18nextify) uses a VDOM implementation - it's like having two VDOM implementation fighting each other...

**Comment by bytenaija** [comment]
> No...the point is...do not use locizify with react...it is really not the use case it was created for.
> 
> Under the hood locizify (using i18nextify) uses a VDOM implementation - it's like having two VDOM implementation fighting each other...

Thank you. 

Is there anyway to use i18next with locizify without having to manually add keys to <Trans i18nKey={key}> for a thousand translation?

**Comment by jamuhl** [comment]
Using react-i18next - using saveMissing - all keys get added like using locizify

Did you already checkout the sample? https://github.com/i18next/react-i18next/tree/master/example/locize

comes with:

- saveMissing
- editor
- ...

**Comment by jamuhl** [comment]
https://github.com/i18next/i18next-cli#instrument might be early but a good alternative
